components/expenses/CreateExpenseForm.js

```diff
@@ -65,21 +66,25 @@ class CreateExpenseForm extends React.Component {
         id: 'expense.error.attachmentMissing',
         defaultMessage: 'Missing attachment',
       },
+      'error.expenseTypeMissing': {
+        id: 'expense.error.expenseTypeMissing',
+        defaultMessage: 'Please pick the type of this expense',
+      },
     });
 
     this.categoriesOptions = categories(props.collective.slug).map(category => {
       return { [category]: category };
     });
 
-    this.expenseTypes = Object.keys(expenseTypes).map(t => {
-      return { [t]: titleCase(t) };
+    this.expenseTypes = Object.entries(expenseTypes).map(t => {
+      return { [t[0]]: titleCase(t[1]) };
```

Comment:
While we're at it, you can destructure `t` as `[key, value]` and use `key` instead of `t[0]` and value instead of `t[1]`